lists/to-release

```diff
@@ -1 +1 @@
-pspy
+ttpassgen
```

Comment:
Don't replace, just add to the list.

Reply:
got it. re added. Thank you for that note.